frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-details-page.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+import { navFactory } from '@console/internal/components/utils';
+
+import { DetailsPage } from '@console/internal/components/factory';
+import { K8sResourceKindReference } from '@console/internal/module/k8s';
+
+import { VMTemplateDetailsFirehose } from './vm-template-details';
+import { TemplateModel } from '../../../../../public/models/index';
+
+export const VMTemplateDetailsPage = (props: VMTemplateDetailsPageProps) => {
+  const pages = [navFactory.details(VMTemplateDetailsFirehose)];
```

Comment:
could we also include the yaml editor?

Reply:
Yes :exclamation: working on it :-)

Reply:
works now, it was just the `exact=true` in the route ...